### PR TITLE
Added 'Please select' option during new account creation

### PR DESCRIPTION
**Pull request by fanxuricky**

### CODE REVIEW

File: src/javascript/binary/common_functions/account_opening.js [hunk 1]
```diff
@@ -162,7 +163,7 @@ const AccountOpening = (() => {
             id;
         $(form_id).find('select, input[type=checkbox]').each(function() {
             id = $(this).attr('id');
-            if (id !== 'tnc') {
+            if (id !== 'tnc' && id !== 'address_state') {
```
Comment (binary-hamedanchi) [inline src/javascript/binary/common_functions/account_opening.js]: What about:
```js
!/^(tnc|address_state)$/.test(id)
```

Comment (fanxuricky) [inline src/javascript/binary/common_functions/account_opening.js]: Is it to replace:
```
id !== 'tnc' && id !== 'address_state'
```

Comment (binary-hamedanchi) [inline src/javascript/binary/common_functions/account_opening.js]: Yes.

Comment (fanxuricky) [inline src/javascript/binary/common_functions/account_opening.js]: Thanks for your suggestion.

**Pull request merged**